cfme/automate/explorer/method.py

```diff
@@ -457,9 +457,7 @@ def create(
 
         add_page = navigate_to(self, 'Add')
 
-        if self.browser.product_version < '5.11':
-            location = location
-        else:
+        if self.browser.product_version > '5.11' and location.islower():
```

Comment:
**Suggestion:**
@ganeshhubale This logic could be like this --> `location.lower() if self.browser.product_version < '5.11' else location.capitalize()`